spec/ValidationAndPasswordsReset.spec.js

```diff
@@ -1082,4 +1082,34 @@ describe('Custom Pages, Email Verification, Password Reset', () => {
         done();
       });
   });
+
+  it('should resolve on an invalid reset password with default server parameters', async () => {
+    await reconfigureServer({
+      appName: 'coolapp',
+      publicServerURL: 'http://localhost:1337/1',
+      emailAdapter: MockEmailAdapterWithOptions({
+        fromAddress: 'parse@example.com',
+        apiKey: 'k',
+        domain: 'd',
+      }),
+    });
+    const result = await Parse.User.requestPasswordReset('test@example.com');
+    expect(result).toEqual({});
+  });
+
+  it('should throw on an invalid reset password', async () => {
```

Comment:
What is the difference between these 2 tests?

Reply:
I forgot to remove the previous test - it's old behaviour